autogpts/autogpt/autogpt/commands/execute_code.py

```diff
@@ -44,6 +66,10 @@
             required=True,
         ),
     },
+    disabled_reason="""To execute python code agent
+    must be running in a Docker container or
+    Docker must be available on the system.""",
+    available=we_are_running_in_a_docker_container() or is_docker_available(),
```

Comment:
This results in unwanted line breaks. Please change this (and the one on line 120) to a regular string.